packages/test/end-to-end-tests/src/test/codePropsal.spec.ts

```diff
@@ -58,6 +58,7 @@ describe("CodeProposal.EndToEnd", () => {
 
     let deltaConnectionServer: ILocalDeltaConnectionServer;
     let opProcessingController: OpProcessingController;
+    let hotSwap = false;
```

Comment:
i you change this to hotSwapContext, you don't need an inline if below, just {hotSwapContext}

Reply:
I intentionally didn't do that so that the tests would cover the common case of options = `undefined` rather than options = `{ hotSwapContext: false }`, gaining a tiny bit of additional coverage. But that is a bit unnecessary, I think.

I changed it.